src/index.html

```diff
@@ -134,8 +132,5 @@ <h4 class="data-text" id="increasing-stairs-container">2 Day Stair Streak</h4>
       </article>
     </section>
   </main>
-  <script src="https://unpkg.com/packery@2.1/dist/packery.pkgd.min.js"></script>
-  <script src="https://unpkg.com/draggabilly@2/dist/draggabilly.pkgd.min.js"></script>
-  <script type="text/javascript" src="src/scripts.js"></script>
```

Comment:
Nice work cleaning this up and getting rid of the scripts and links that are not being used now due to webpack!